### PR TITLE
Added --remove-deactivated-users flag to channel move

### PR DESCRIPTION
**Pull request by icelander**
This was added in [5.4](https://mattermost.atlassian.net/browse/MM-12251) but wasn't added to the docs

Signed-off-by: Paul Rothrock <paul@movetoiceland.com>

### CODE REVIEW

File: source/administration/command-line-tools.rst [hunk 1]
```diff
@@ -299,6 +299,7 @@ mattermost channel move
     .. code-block:: none
 
           --username [REQUIRED] Username of the user who is moving the team.
+          --remove-deactivated-users [OPTIONAL] When moving the channel, remove any users who have been deactivated who may be preventing the move
```
Comment (amyblais) [inline source/administration/command-line-tools.rst]:
```suggestion
          --remove-deactivated-users [OPTIONAL] When moving the channel, remove any users who have been deactivated who may be preventing the move.
```

**Comment by jasonblais** [comment]
@icelander Can you help review the above feedback from Amy and Katie? Thanks!

**Comment by justinegeffen** [comment]
@icelander, wanted to check in on this and find out whether you've had a chance to review the feedback. :)

**Pull request merged**